server.js

```diff
@@ -7,7 +7,9 @@ const prettyBytes = require('pretty-bytes');
 const queryString = require('query-string');
 const semver = require('semver');
 const xml2js = require('xml2js');
+const Raven = require('raven');
 
+Raven.config().install();
```

Comment:
Could you add a parameter so that we can default to private/secret.json?

You only need to move this line down and make it:

```js
Raven.config(process.env.SENTRY_DSN || serverSecrets.sentry_dsn).install();
```

Maybe we can merge lib/server-secrets.js and lib/server-config.js in a follow-up? Wdyt @paulmelnikow?

Reply:
Good idea. I've moved `const serverSecrets = require('./lib/server-secrets');` above the `Raven.config()...`. I want to have Sentry working as soon as possible.